CRM/Mailing/BAO/Mailing.php

```diff
@@ -1644,7 +1644,7 @@ public static function create(array $params) {
     // In v4 of the api they are not available via CRUD. At some
     // point we will create a 'submit' function which will do the crud+submit
     // but for now only CRUD is available via v4 api.
-    if (($params['version'] ?? '') !== 4) {
+    if (empty($params['skip_legacy_scheduling'])) {
       self::doSubmitActions($params, $mailing);
```

Comment:
@eileenmcnaughton  Is `skip_legacy_scheduling` different from `_skip_evil_bao_auto_schedule_`? (See first line of `doSubmitActions()`.)

Reply:
@totten the 2 skip evils are subsets of skip legacy